desktop/components/react/utils/render_layout.js

```diff
@@ -0,0 +1,36 @@
+import renderTemplate from 'desktop/components/react/utils/render_template'
```

Comment:
Yeah, right? If you twist the cube far enough..... 😄 

This is a typical way of determining whether one is dealing with a Class or a SFC, but still doesn't quite get us to where exactly we need to be (is this a string-renderable component?):

```javascript
function isStateless(Component) {
    return !Component.prototype.render;
}
```
Let me investigate a bit more and add a check. Haha got a bit excited to show you this and probably hit the send button too early.